process/pfcoil.py

```diff
@@ -2973,10 +2973,8 @@ def deltaj_hijc_rebco(temperature):
 
             # Actual current density in superconductor, which should be equal to jcrit(thelium+tmarg)
             # when we have found the desired value of tmarg
-            lap = 0
-            while True:
-                lap = lap + 1
-                if (ttest <= 0.0) or (lap > 100):
```

Comment:
Again because of the > my checks are showing this loop might run one less than before (in practice it doesn't matter), but I think retaining the same behaviour would be ideal. Let me know if im being silly

Reply:
it will run from 0 -> 99 (100 runs) and previously because immediately `lap = 1`  this will run from 1 to 100 (100 runs) and then on the 101st immediately break.

I think thats right...

Reply:
eg
```python
l = 0

while True:
    l += 1
    if l > 5:
        break
    print("hi")
```
will print 5 'hi's

Reply:
Yep, logic is after the `break` on this one, you're right